pkg/util/util.go

```diff
@@ -135,6 +135,9 @@ func GetHostname() string {
 	if err != nil {
 		return constants.DefaultHostnameOverride
 	}
+	// Hostname should be normailzied because it's case-insensitive
```

Comment:
Thx for your contributions!  Can we call `GetHostName` func of kubernetes mentioned in the issue https://github.com/kubeedge/kubeedge/issues/5391 here?   In this way, if there is any change in Kubernetes, it can be directly synchronized.

Reply:
Yes, it's the better implementation.